src/components/RoundButton/RoundButton.jsx

```diff
@@ -1,24 +1,20 @@
 import PropTypes from "prop-types";
-import "./RoundButton.css";
+import { Button } from "./styles";
 
 const RoundButton = ({ children, variant, isActive, onClick }) => {
-  const buttonClasses = [
-    "round-button",
-    `round-button__${variant}`,
-    isActive ? "round-button--active" : "",
-  ].join(" ");
-
   return (
-    <button onClick={onClick} className={buttonClasses}>
+    <Button $variant={variant} $isActive={isActive} onClick={onClick}>
```

Comment:
Variant will indeed leak, but `isActive` is not valid, so you can remove the transient

```suggestion
    <Button $variant={variant} isActive={isActive} onClick={onClick}>
```